app/server/server.js

```diff
@@ -1,17 +1,21 @@
 import express from 'express';
 import 'dotenv/config';
+import session from "express-session";
+import jwt from "jsonwebtoken";
```

Comment:
be consistent with quotations


```suggestion
import session from 'express-session';
import jwt from 'jsonwebtoken';
```